### PR TITLE
Add support for reserved_internal_range in net-vpc

### PR DESCRIPTION
**Pull request by sruffilli**
[Provider version 5.29.0](https://github.com/hashicorp/terraform-provider-google/releases/tag/v5.29.0) introduces [reserved_internal_range](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_subnetwork#reserved_internal_range), which triggered an unexpected error during plan (potentially due to secondary_ip_ranges being previously implemented with [attr-as-blocks](https://developer.hashicorp.com/terraform/language/attr-as-blocks)).

This PR adds explicit support for reserved_internal_range in net-vpc.

### CODE REVIEW

**Comment by juliocc** [comment]
are you intentionally removing private_nat subnet support?

**Comment by ludoo** [comment]
> are you intentionally removing private_nat subnet support?

Yowz I missed that 🤣

File: modules/net-vpc/variables.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2024 Google LLC
+ * Copyright 2023 Google LLC
```
Comment (juliocc) [inline modules/net-vpc/variables.tf:2]: 2024

**Review by juliocc — CHANGES_REQUESTED** [review]

**Review by ludoo — CHANGES_REQUESTED** [review]
Reverting approval to prevent automerge

**Comment by ludoo** [comment]
Probably based on a non up to date master version

**Comment by juliocc** [comment]
> Reverting approval to prevent automerge

tests caught it anyway :)

**Comment by sruffilli** [comment]
Great catch Julio!

download

**Review by ludoo — APPROVED** [review]

**Pull request merged**